### ISSUE TITLE
Release delphi_utils on PyPI

### ISSUE THREAD

**Issue by krivard** [cmu-delphi/covidcast-indicators#995]
We need to release `delphi_utils` publicly to unblock some of @dshemetov 's work on non-numeric encoding across indicators and delphi-epidata.

The rough steps are these:

- [ ] Create a LICENSE file in `_delphi_utils`. Use the same kind of [MIT License we use for delphi-epidata](https://github.com/cmu-delphi/delphi-epidata/blob/main/LICENSE)
- [ ] Update `_delphi_utils/README.md` to be appropriate for a non-Delphi-Engineering audience. This README will become part of the materials posted on PyPI. It should include a short list of the submodules and their function, as well as a link back to this repository. For reference, the [delphi-epidata README](https://github.com/cmu-delphi/delphi-epidata/blob/main/src/client/packaging/pypi/README.md) is pretty sparse; so is the [covidcast README](https://github.com/cmu-delphi/covidcast/blob/main/Python-packages/covidcast-py/README.md), so we don't need to be completionists here.
- [ ] Follow the [rest of the packaging & uploading instructions](https://packaging.python.org/tutorials/packaging-projects/). Have @korlaxxalrok create a dedicated Delphi account and tokens on PyPI and Test PyPI for this purpose, and store the credentials somewhere secure. We'll deal with adding this account to delphi-epidata and covidcast when I get back.  
- [ ] 🎉 

Delegate as needed!

**Comment by chinandrew** [comment]
do we want all the indicators to start installing utils from pip? upside is they're all standardized, downside is we have a bit more involved release process for changes.

**Comment by chinandrew** [comment]
released. will open a separate issue re: existing indicators
